src/widgets/Container/index.js

```diff
@@ -3,17 +3,33 @@ import Markdown from '../Markdown';
 import styles from './styles.css';
 
 export default class Container extends React.PureComponent {
+
+  getViewSource(schema) {
```

Comment:
Hm, actually, I don't think that the presence or absence of $id/id matters here -- we should never be linking to an id. So I think this can all be reduced to just a show/hide source link without referencing `schema.$id` or `schema.id` at all.

Reply:
Ok. if we are never going to be linking to an id, then I guess this can be removed

Reply:
Since we are showing the whole json in json view, and $id will be part of the json.
Do you want me to add the $id in title also when we are in json view?